js.go

```diff
@@ -146,10 +146,30 @@ type JSOpt func(opts *js) error
 
 func ApiPrefix(pre string) JSOpt {
 	return func(js *js) error {
-		js.pre = pre
-		if !strings.HasSuffix(js.pre, ".") {
-			js.pre = js.pre + "."
+		if strings.ContainsAny(pre, " \t\r\n*") {
```

Comment:
So this prevent a subject like `foo.b*r.baz`?

Reply:
Yeah. I added that case now.
So we prevent these:
```go
{
	prefix:  "foo.*",
	wantErr: true,
},
{
	prefix:  "foo.b*r.baz",
	wantErr: true,
},
```